Setting Up Neolink For Use With Blue Iris.md

```diff
@@ -0,0 +1,125 @@
+# Setting Up Neolink For Use With Blue Iris
+
+![CI](https://github.com/thirtythreeforty/neolink/workflows/CI/badge.svg)
+
+Stock Reolink cameras do not support use with non-proprietary surveillance software such as Blue Iris. Neolink enables you to use your unsupported cameras with Blue Iris and other viewers/recorders. In this guide, you will learn how to configure your Reolink cameras and Neolink for use with Blue Iris software.
+
+**This guide applies to the following camera models:**
+- B800/D800
+- E1
+
+
+## Step One: Configuring Your Cameras
+The first thing to do to make video recording run smoothly is tweak the settings on your Reolink Cameras so that they do not conflict with Neolink.
```

Comment:
"do not conflict" sounds adversarial... How about "tweak the settings on your cameras to work around known Neolink problems, and to set them up as a part of your network."